src/sonic-host-services/scripts/hostcfgd

```diff
@@ -987,18 +1013,25 @@ class HostConfigDaemon:
         # Initialize AAACfg
         self.hostname_cache=""
         self.aaacfg = AaaCfg()
-        
 
-    def load(self):
-        aaa = self.config_db.get_table('AAA')
-        tacacs_global = self.config_db.get_table('TACPLUS')
-        tacacs_server = self.config_db.get_table('TACPLUS_SERVER')
-        radius_global = self.config_db.get_table('RADIUS')
-        radius_server = self.config_db.get_table('RADIUS_SERVER')
+    def load(self, init_data):
+        features = init_data['FEATURE']
+        aaa = init_data['AAA']
+        tacacs_global = init_data['TACPLUS']
+        tacacs_server = init_data['TACPLUS_SERVER']
+        radius_global = init_data['RADIUS']
+        radius_server = init_data['RADIUS_SERVER']
+        lpbk_table = init_data['LOOPBACK_INTERFACE']
+        ntp_server = init_data['NTP_SERVER']
+        ntp_global = init_data['NTP']
+
+        self.feature_handler.update_all_features_config(features)
         self.aaacfg.load(aaa, tacacs_global, tacacs_server, radius_global, radius_server)
+        self.iptables.load(lpbk_table)
+        self.ntpcfg.load(ntp_global, ntp_server)
 
         try:
-            dev_meta = self.config_db.get_table('DEVICE_METADATA')
+            dev_meta = init_data['DEVICE_METADATA']
```

Comment:
>init_data [](http://example.com/codeflow?start=23&length=9)

`DEVICE_METADATA` is no subscribed, so there will be no `init_data`.

And the `except Exception as e` is evil, which swallow this critical bug.

Reply:
I would love to remove this error handling but I am not sure what it was intended to handle :( 

`exception Exception` + no comments is bad

I can fix this though, thanks for the catch.

Reply:
To me, no exception is expected. Let's just remove `try-except`.